src/main/java/se/liu/ida/hefquin/engine/queryplan/physical/impl/PhysicalOpBindJoin.java

```diff
@@ -58,4 +59,17 @@ public void visit(final PhysicalPlanVisitor visitor) {
         visitor.visit(this);
     }
 
+    @Override
+    public String toString() {
+        final StringBuilder builder = new StringBuilder();
+
+        if ( lop instanceof LogicalOpTPAdd)  {
+            builder.append( "> bindJoin");
+            return ( (LogicalOpTPAdd) lop ).toString();
```

Comment:
That does not work as expected. You are not using the builder in the return statement.

Reply:
The same issue repeats in the other physical operators as well.